docs/en/observability/whats-new.asciidoc

```diff
@@ -1,4 +1,4 @@
-[[whats-new]]
+hance[[whats-new]]
```

Comment:
Typo?

Reply:
whoops, was probably in the doc when I thought I was in find.